src/pages/Signin/Signin.js

```diff
@@ -1,5 +1,67 @@
 import React from 'react';
+import styled from 'styled-components';
 
 export default function Signin() {
-  return <div>Signin</div>;
+  return (
+    <div>
+      <Container>
+        <JoinWrap>
+          <JoinBox>
+            <Title>회원가입</Title>
+            <KakaoBtnImg src="images/main/kakao_login_medium_wide.png"></KakaoBtnImg>
```

Comment:
```suggestion
            <KakaoBtnImg src="images/main/kakao_login_medium_wide.png" />
```

셀프 클로징 해주시고 alt 속성이 빠졌습니다! 추가해주세요!